pset-functions.js

```diff
@@ -208,6 +260,20 @@ console.log('Problem 5 - bonus:')
 
 // Add your code below this line
 
+const wordCount = (singleWord) => {
+    const words = singleWord.split(" ")
+    const oneWord ={}
+
+    words.forEach((eachWord) => {
+        if (oneWord[eachWord] === undefined) {
```

Comment:
This is very well done; you are inserting key/value pairs into the empty object if the word does not already exist. Great use of object knowledge, forEach method, if/else statements and object bracket notation. 

Sammi, you did a really awesome job on this homework. Really clean code and well thought out solutions. Keep up the really good work!